### PR TITLE
USHIFT-503: Document unidle service known limitation

### PR DESCRIPTION
**Pull request by oglok**
Signed-off-by: Ricardo Noriega <rnoriega@redhat.com>

**Which issue(s) this PR addresses**:

Closes [USHIFT-503](https://issues.redhat.com/browse/USHIFT-503)

### CODE REVIEW

**Comment by zshi-redhat** [comment]
nit: do we want a reference link in the top-level README?

File: docs/known_limitations.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Known Limitations
+
+Despite the list of enabled APIs is documented, there might be some cases where it's not clear whether a certain functionality works. This is a list of known limitations within MicroShift:
+
+## Idling/Unidling Applications
+
+The `oc` client provides a way to idle applications in order to reduce power consumption. By adding specific annotations, `oc` will scale all workloads referenced to a service down to zero. This is a fully working feature in MicroShift.
+
+However, when the application service/route receives network traffic, OVN-Kubernetes will detect it and write an annotation to scale that application back up. MicroShift does not provide the unidling controller though, so this functionality won't work.
```
Comment (zshi-redhat) [inline docs/known_limitations.md:9]: OVN-Kubernetes sends an Normal type event to scale ...

Comment (zshi-redhat) [reply docs/known_limitations.md:9]: https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/controller/unidling/unidle.go#L207

Comment (oglok) [reply docs/known_limitations.md:9]: Good catch, idling is annotation based, so wrong assumption from my side.

**Pull request merged**